.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ templates:
         description: "resource type for underlying VM"
         default: "medium"
         type: enum
-        enum: ["small", "medium", "large", "xlarge"]
+        enum: ["small", "medium", "medium+", "large", "xlarge"]
```

Comment:
Seems like `medium+` is never used, so why add it?

Reply:
We don't use any of the sizes besides medium currently, but since `medium+` is available it seemed logical to add it.